keras_nlp/src/utils/preset_utils.py

```diff
@@ -546,7 +546,12 @@ def load_config(preset, config_file=CONFIG_FILE):
     return config
 
 
-def validate_metadata(preset):
+def check_format(preset):
+    if check_file_exists(preset, "model.safetensors") or check_file_exists(
```

Comment:
I think we've been keeping all these filenames as constants at the top of the file, can probably do the same here.

Reply:
Noted!